integrations/mixpanel/.buildscript/licenses/android-sdk-license

```diff
@@ -0,0 +1 @@
+d56f5187479451eabf01fb78af6dfcb131a6481e
```

Comment:
The android sdk license also supported a manual release process, which is now deprecated, so we shouldn't need this file.